sdk/tests/conformance2/textures/misc/tex-mipmap-levels.html

```diff
@@ -59,6 +59,28 @@
     gl_FragColor = texture2D(tex, texCoord);
 }
 </script>
+
+<script id="vshader_texsize" type="x-shader/x-vertex">#version 300 es
+uniform vec4 uMult;
```

Comment:
Yes, it can be omitted. It was just because of the existing code around. Fixed anyway!